### PR TITLE
OCD-3586 - Update README documentation for each project

### PR DESCRIPTION
**Pull request by tmy1313**
OCD-3586

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,13 +35,37 @@ chpl/chpl-service/src/test/resources/environment.test.properties
 
 ## Properties files
 
-Copy over the two template files and fill in the keylocation, downloadedFolderPath, and datasourcepassword
+The following Proiperties Files exist within the CHPL Resources project.
```
Comment (andlar) [inline README.md]:
```suggestion
The following Properties Files exist within the CHPL Resources project.
```

File: README.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ The CHPL api
 ## Install required software
 
 Java 1.8.0
-Postgres 9.4.4
+Postgres 10.15
 mvn 3.3.3
-Tomcat 8.0.24
+Tomcat 8.5.64
 Eclipse (latest)
```
Comment (andlar) [inline README.md]:
```
* Java 1.8.0
* PostgreSQL 10.15
* mvn 3.3.3
* Tomcat 8.5.x --whatever the latest version is
* Eclipse (latest)
```

The text in the document looks like it should be a bulleted list, but it shows up on the page as a single line. The suggested change in this comment would make it a bulleted list
image

File: README.md [hunk 3]
```diff
@@ -35,13 +35,37 @@ chpl/chpl-service/src/test/resources/environment.test.properties
...
+chpl/chpl-resources/src/main/resources/errors.properties
+chpl/chpl-resources/src/main/resources/lookup.properties
 ```
+Any value in the above files can be overridden by creating an "override" file.  The file should be name `originalFileName-override.properites` and be placed somewhere on the classpath.  In the "override" file, you can redefine any existing key with a new value.
+
+Each of the above files has "default" values for each key, though a few keys within the files have a value of "SECRET".  This applies to certain values that only apply to a particular instance of CHPL - things like server names, usernames, passwords, etc.
+
+| File                   | Key                                | Description                                                                                       |
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+| email.properties       | directReview.chplChanges.email     |                                                                                                   |
```
Comment (andlar) [inline README.md]:
```suggestion
| File                   | Key                                | Description                                                                                       |
|---|---|---|
| email.properties       | directReview.chplChanges.email     |                                                                                                   |
```

**Review by andlar — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -35,13 +35,37 @@ chpl/chpl-service/src/test/resources/environment.test.properties
...
+chpl/chpl-resources/src/main/resources/errors.properties
+chpl/chpl-resources/src/main/resources/lookup.properties
 ```
+Any value in the above files can be overridden by creating an "override" file.  The file should be name `originalFileName-override.properites` and be placed somewhere on the classpath.  In the "override" file, you can redefine any existing key with a new value.
+
+Each of the above files has "default" values for each key, though a few keys within the files have a value of "SECRET".  This applies to certain values that only apply to a particular instance of CHPL - things like server names, usernames, passwords, etc.
+
+| File                   | Key                                | Description                                                                                       |
+|------------------------|------------------------------------|---------------------------------------------------------------------------------------------------|
+| email.properties       | directReview.chplChanges.email     |                                                                                                   |
```
Comment (kekey1) [inline README.md]: Email account to send notification of listing or developer changes that do affect direct reviews

File: README.md [hunk 5]
```diff
@@ -35,13 +35,37 @@ chpl/chpl-service/src/test/resources/environment.test.properties
...
+
+Each of the above files has "default" values for each key, though a few keys within the files have a value of "SECRET".  This applies to certain values that only apply to a particular instance of CHPL - things like server names, usernames, passwords, etc.
+
+| File                   | Key                                | Description                                                                                                  |
+|------------------------|------------------------------------|--------------------------------------------------------------------------------------------------------------|
+| email.properties       | directReview.chplChanges.email     | Email address to be notified when changes are made to CHPL entities that involve Direct Reviews              |
+| email.properties       | directReviewunknownChanges.email   | Email address to be notified when changes are made to CHPL entities that possibley involve Direct Reviews    |
```
Comment (kekey1) [inline README.md]:
```suggestion
| email.properties       | directReviewunknownChanges.email   | Email address to be notified when changes are made to CHPL entities that possibly involve Direct Reviews    |
```

**Review by kekey1 — CHANGES_REQUESTED** [review]

**Review by kekey1 — DISMISSED** [review]

File: README.md [hunk 6]
```diff
@@ -35,13 +35,37 @@ chpl/chpl-service/src/test/resources/environment.test.properties
...
 ```
-chpl/chpl-auth/src/main/resources/environment.auth.properties
-
-chpl/chpl-service/src/main/resources/environment.properties
+chpl/chpl-resources/src/main/resources/email.properties
+chpl/chpl-resources/src/main/resources/environment.properties
+chpl/chpl-resources/src/main/resources/errors.properties
+chpl/chpl-resources/src/main/resources/lookup.properties
 ```
+Any value in the above files can be overridden by creating an "override" file.  The file should be name `originalFileName-override.properites` and be placed somewhere on the classpath.  In the "override" file, you can redefine any existing key with a new value.
```
Comment (andlar) [inline README.md]:
```suggestion
Any value in the above files can be overridden by creating an "override" file.  The file should be name `originalFileName-override.properties` and be placed somewhere on the classpath.  In the "override" file, you can redefine any existing key with a new value.
```

**Review by andlar — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Review by palakpatel311 — APPROVED** [review]

**Pull request merged**